plugins/plugin-radius/src/components/recipes/RecipeTable.tsx

```diff
@@ -41,12 +41,32 @@ export const RecipeTable = ({
     { title: 'Template Path', field: 'templatePath' },
   ];
 
+  const filters: TableFilter[] = [
+    {
+      column: 'Name',
+      type: 'select',
+    },
+    {
+      column: 'Type',
+      type: 'multiple-select',
+    },
+    {
+      column: 'Kind',
+      type: 'multiple-select',
+    },
+    {
+      column: 'Template Path',
+      type: 'multiple-select',
+    },
+  ];
+
   return (
     <Table
       title={title || 'Recipes'}
```

Comment:
The title makes it redundant since we have the page title set already. Therefore we can remove this. I will make sure to remove the redundant title from a couple of other pages too.

Reply:
Done